wordpress-mobile/WordPress-iOS
# Save for later: Implement "View all" in Notice when marked as Saved

Implements #9399 

Implements "View All" button in notice presented when a post is marked as Saved for Later

![view_all_ipad mov](https://user-images.githubusercontent.com/2722505/40285482-58d1f872-5c51-11e8-84c6-77bc11ed90fb.gif)
![view_all_8 mov](https://user-images.githubusercontent.com/2722505/40285483-5e5c1c46-5c51-11e8-969e-804eca276d10.gif)

I am not a huge fan of posting notifications, but in this case, this seemed like the only way forward. 

To test:
0. Activate the feature flag
1. Navigate to Reader
2. Navigate to Followed Sites
3. Mark a post as Saved for Later.
4. Tap View All in the Notice that is presented.
